ambari/administration/admin-tasks.rst

```diff
@@ -0,0 +1,109 @@
+Ambari Admin Tasks
+==================
+
+An "Ambari Admin" has administrator (or super-user) privilege. When logged into Ambari with the "Ambari Admin" privilege, you can:
+
+* :ref:`Change the Administrator Account Password<ambari-change-admin-password>`
+* :ref:`Create a cluster<ambari-creating-cluster>`
+* :ref:`Set access permissions for an existing cluster<ambari-cluster-permission>`
+* :doc:`Create, delete, and edit view instances</ambari/administration/views>`
+* :ref:`Manage permissions for view instances<ambari-setting-view-permissions>`
+* :doc:`Create, edit, and delete users and user groups</ambari/administration/user-group>`
+
+For more information about creating Ambari users locally and importing Ambari LDAP users, see :doc:`Managing Users and Groups</ambari/administration/user-group>`.
+
+
+.. _ambari-change-admin-password:
+
+Changing the Administrator Account Password
+___________________________________________
+
+During install and setup, the Cluster Installer wizard automatically creates a default user with "Ambari Admin" privilege. You can change the password for this user (or other Local users in the system) from the Ambari Administration interface. You can change the password for the default admin user to create a unique administrator credential for your system.
+
+To change the password for the default admin account:
+
+#. Browse to the Users section.
+#. Select the admin user.
+#. Click the Change Password button.
+#. Enter the current admin password and the new password twice.
+#. Click OK to save the new password.
+
+.. _ambari-creating-cluster:
+
+Creating a Cluster
+__________________
+
+As an Ambari Admin, you can launch the Cluster Install Wizard and create a cluster. To create a cluster, from the Ambari Administration interface:
+
+#. Click ``Install Cluster``. The Cluster Install Wizard displays.
+#. Follow the steps in the wizard to install your cluster.
+
+For more information about prerequisites and system requirements, see :doc:`Getting Ready</installation/getting-ready/index-getting-ready>`.
+
+.. _ambari-cluster-permission:
+
+Setting Cluster Permissions
+___________________________
+
+After you create a cluster, users with Admin Admin privileges automatically get Operator permission on the cluster. By default, no users have access to the cluster. You can grant permissions on the cluster to other users and groups from the Ambari Administration interface.
+
+Ambari manages the following permissions for a cluster: ``Operator`` and ``Read-Only``. Users and Groups with ``Operator`` permission are granted access to the cluster. Operator permission provides full control of the following services:
+
+* Start
+* Stop
+* Restart
+* Add New
+
+And the following configurations:
+
+* Modify
+* Revert
+
+Users and Groups with ``Read-Only`` permission can only view, not modify, services and configurations.
```

Comment:
What about _Users and Groups with Read-Only permission can only view services and configurations; they cannot alter them._?